spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -1906,7 +1907,7 @@ private void wrapUp(@Nullable Throwable throwable) {
 				this.consumerSeekAwareListener.onPartitionsRevoked(partitions);
 				this.consumerSeekAwareListener.unregisterSeekCallback();
 			}
-			this.logger.info(() -> getGroupId() + ": Consumer stopped");
+			this.logger.info(() -> KafkaMessageListenerContainer.this.getGroupId() + ": Consumer stopped");
```

Comment:
Why don't use that `this.consumerGroupId` instead?

Reply:
There is no reason other than to preserve the existing get method. But I think it is okay to use `this.consumerGroupId` as it is initialized when creating the ListenerConsumer and can't be changed. 

I did the same for the override method `getGropuId()` ([link](https://github.com/spring-projects/spring-kafka/pull/3318/files#diff-49bc63455f16e2f0b1fe17ed1ec885135d494883d5a21a3da53d031098d09a33R3333)). Thanks!